hugo-annotations/src/main/java/hugo/weaving/DebugLog.java

```diff
@@ -7,7 +7,9 @@
 import static java.lang.annotation.ElementType.METHOD;
 import static java.lang.annotation.ElementType.TYPE;
 import static java.lang.annotation.RetentionPolicy.CLASS;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
-@Target({TYPE, METHOD, CONSTRUCTOR}) @Retention(CLASS)
+@Target({TYPE, METHOD, CONSTRUCTOR}) @Retention(RUNTIME)
 public @interface DebugLog {
+    boolean trace() default false;
```

Comment:
As I said in the original issue, I don't think this should be an option. We should always add trace markers when the annotation is used.